### PR TITLE
Add OnVehicleDataUpdated test

### PR DESCRIPTION
**Pull request by Kozoriz**
Test only for add coverage,
	because this method dont do anything.

Please review : @anosach-luxoft , @dev-gh , @VProdanov , @LevchenkoS, @okozlovlux

### CODE REVIEW

**Comment by LevchenkoS** [comment]
@Kozoriz Reviewed.

File: src/components/application_manager/test/policy_handler_test.cc [hunk 1]
```diff
@@ -1476,6 +1476,16 @@ TEST_F(PolicyHandlerTest, CanUpdate) {
   EXPECT_TRUE(policy_handler_.CanUpdate());
 }
 
+TEST_F(PolicyHandlerTest, OnVehicleDataUpdated) {
+  // Arrange
+  ChangePolicyManagerToMock();
+
+  // Act
+  ::smart_objects::SmartObject message_(::smart_objects::SmartType_Map);
```
Comment (okozlovlux) [inline src/components/application_manager/test/policy_handler_test.cc]: @Kozoriz why underscore in name **message**?

**Comment by dev-gh** [comment]
Reviewed

File: src/components/application_manager/test/policy_handler_test.cc (hunk 1 above)
Comment (Kozoriz) [inline src/components/application_manager/test/policy_handler_test.cc]: @okozlovlux , corrected 
	8d7f021

**Comment by okozlovlux** [comment]
@Kozoriz reviewed

**Comment by VProdanov** [comment]
Reviewed

**Comment by anosach-luxoft** [comment]
Reviewed

**Pull request merged**